test/error-handler/try-and-call-abstract-function.js

```diff
@@ -1,6 +1,7 @@
 // additional functions
 // abstract effects
-// expected errors: [{location: {"start":{"line":26,"column":11},"end":{"line":26,"column":21},"identifierName":"abstractFn","source":"test/error-handler/try-and-call-abstract-function.js"}, errorCode: "PP0021", severity: "RecoverableError", message: "Possible throw inside try/catch is not yet supported"}]
+// expected errors: [{location: {"start":{"line":27,"column":11},"end":{"line":27,"column":21},"identifierName":"abstractFn","source":"test/error-handler/try-and-call-abstract-function.js"}, errorCode: "PP0021", severity: "RecoverableError", message: "Possible throw inside try/catch is not yet supported"}]
+// recover-from-errors
```

Comment:
What does this do?

Reply:
It is meta data for the test runner, I've had to change the line number to 27 because I've added the `// recover-from-error` which tells is that the behaviour of this test has changed now that we expect to recover from such abstract cases .

Reply:
This change still triggers an error to be logged and which needs to be explicitly recovered from.